pmm-app/src/pmm-qan/panel/components/Filters/components/CheckboxGroup/CheckboxGroup.tsx

```diff
@@ -56,7 +56,7 @@ export const CheckboxGroup: FC<CheckboxGroupProps> = ({
             />
           </span>
           <span className={styles.percentage}>
-            <span>{valueExists ? humanize.transform(item.main_metric_percent, 'percent') : null}</span>
+            <span>{valueExists ? humanize.transform(item.main_metric_percent, 'percentRounded') : null}</span>
```

Comment:
It shouldn't be rounded here because percentage values may be relly small and we need to differentiate it somehow